### DISCUSSION TITLE
stories_things/basic_content/using/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#14] [Announcements]
# stories_things/basic_content/using/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/using/

**Comment by cxz0218** [comment]
大佬~，改变访问性的cpp文件中，
class Derived : Base {
public:
  using Base::value;  // error: 'int Base::value' is private within this context
  void test2() { cout << "value is " << value << endl; }
};
value在基类的private，所以using Base::value;这里应该是error叭？

**Reply by Light-City** [reply]
是的，这里不是有注释吗

**Reply by huizi-sui** [reply]
cpp里没有注释啊
